app/src/main/res/values/strings.xml

```diff
@@ -124,7 +124,7 @@
     <string name="history">History</string>
 
     <!-- Below values need to be updated before each launch -->
-    <string name="encryption_key">hi</string>
-    <string name="encryption_salt">you</string>
-    <string name="google_maps_key">guys!</string>
+    <string name="encryption_key">a</string>
```

Comment:
are we deprecating this for a `keys.properties` file?

Reply:
if not, let's create a git issue

Reply:
ye ill create an issue

Reply:
#198